Source/Python/WarpXWrappers.cpp

```diff
@@ -212,6 +212,13 @@ extern "C"
       ConvertLabParamsToBoost();
     }
 
+    void warpx_CheckGriddingForRZSpectral()
+    {
+#if (defined WARPX_DIM_RZ) && (defined WARPX_USE_PSATD)
+      CheckGriddingForRZSpectral();
+#endif
```

Comment:
Could we remove the macros here? They are already present in the function, so let's help readability here.

Reply:
Ok